libraries/engage/product/selectors/product.js

```diff
@@ -76,3 +76,96 @@ export function makeGetProductFeaturedMedia() {
     }
   );
 }
+
```

Comment:
some of the functions here do exist in commerce/product/selectors/product.
please import them from there